### ISSUE TITLE
When the keyboard focus moves to the "select subscription" list, JAWS/NVDA announces 'type to narrow down result' and the list name: A11y_Azure Machine Learning Extension for VS Code_Appling filter on subscription_Screen reader

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/vscode-tools-for-ai#2489]
**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**
"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

## GitHub Tags
#A11yMAS; #A11yTCS; #Win32; #DesktopApp; #A11ySev3; #BM_Azure Machine Learning Extension for VS Code_Win32_Jan2024; #Azure Machine Learning Extension for VS Code; #WCAG1.3.1; #FTP; #Screen Reader; #NVDA; #JAWS; #Element:List; #STP; #External; #External:VSCode; #FirstParty:Bug245220; #Regressed:04-01-24;

## Environment Details:
Application: Azure Machine Learning Extension for VS Code
Version: 1.2.0
Visual studio code version: 1.96.3
OS: Windows 11 Enterprise 24H2
Build: 26100.2605
Screen reader
JAWS Version: 2024.2312.53
NVDA Version: 2024.4.1

## Repro Steps

1. Turn on Jaws/NVDA screen reader
2. Launch the VS Code.
3. Install the Azure Machine Learning extension and sign in with appropriate Credentials.
4. Tab until the left toolbar (list of icons). This toolbar has a file icon, search icon, settings, etc.
5. Use the up/down arrow keys to navigate to the "Azure" icon and press enter
6. The Azure panel will now be active next to the toolbar.
7. Tab until the Machine Learning dropdown.
8. Tab till any subscription for example AML V1 Personal subscription.
9. Tab until the "Filter" button and press Enter. It will be a button with a funnel icon.
10. TAB till "Select subscriptions" combo box.
11. TAB till "Select subscriptions" list.
12. Observer that when the keyboard focus moves to the select subscription list, JAWS/NVDA announces, 'type to narrow down result' and the list name.

## Actual Experience
When the keyboard focus moves to the select subscription list, JAWS/NVDA announces, 'type to narrow down result' and the list name. This will miss lead the screen reader user because keyboard focus is on list not on any edit filter.  

### Similar issue observed with following scenarios:

**Scenario1**: Creating a job Select an Azure ML job Schema

1. TAB till create job button and press ENTER key
2. TAB till list and observe the issue.

**Scenario2**: Setting Default Workspace

1. Open the Command Palette via the keyboard command Ctrl + Shift + P 
2. Type "Set default workspace" and select the option from the dropdown list called "Azure ML: Set Default Workspace") and press enter
3. A list should appear that says "Select a subscription"
4. Use the up/down arrows to navigate the list until "AML V1 Personal 2" is selected, and press enter
5. A list should appear that says "Select a workspace"
6. TAB till list and observe the issue.

## Expected Experience
When the keyboard focus moves to the select subscription list, JAWS/NVDA should announce the list name only. 

## User Impact:
Screen reader user will miss lead as focus is not on any edit filter, but JAWS/NVDA announces, 'type to narrow down result'.

## Attachment
NVDA:

![Image](https://github.com/user-attachments/assets/404415a2-6c14-47e4-8311-340209a15ada)

JAWS:

![Image](https://github.com/user-attachments/assets/b1820bf1-624a-4047-a389-685e7b38464a)

Code snippet

![Image](https://github.com/user-attachments/assets/3102febc-d6f7-4346-bb71-c0dcbf7cc030)

Similar issue of scenario 1 and 2

![Image](https://github.com/user-attachments/assets/b6f22fe7-d526-4b18-9685-27b5ae2c2e29)

**Comment by JerryYangKai** [comment]
Hello @kapilvaishna  
I cannot fix it, the reason is similar with this one https://github.com/microsoft/vscode-tools-for-ai/issues/2491
This experience is by design from VS Code side, All the VS Code extension need to follow this UI guideline and cannot change it.

**Comment by kapilvaishna** [comment]
External bug: https://github.com/microsoft/vscode/issues/245220

**Comment by Yash14j** [comment]
GithubTags:#ResolvedByTT

**Comment by kapilvaishna** [comment]
Downgraded to sev3
